### PR TITLE
Add an `update` command to the CLI that introspects the database.

### PR DESCRIPTION
**Pull request by SamirTalwar**
### What

This works similarly to the update behavior of the old configuration server.

### How

There wasn't a lot to do.

This does not update the scripts; I will follow up with that later.

### CODE REVIEW

File: crates/cli/tests/update_tests.rs [hunk 1]
```diff
@@ -0,0 +1,42 @@
+use std::fs;
+
+use ndc_postgres_cli::*;
+use ndc_postgres_configuration as configuration;
+use ndc_postgres_configuration::RawConfiguration;
+
+const CONNECTION_STRING: &str = "postgresql://postgres:password@localhost:64002";
+
+#[tokio::test]
+async fn test_update_configuration() -> anyhow::Result<()> {
```
Comment (soupi) [inline crates/cli/tests/update_tests.rs:10]: When does this run? As part of `just dev` and `just test`?

**Review by soupi — APPROVED** [review]
LGTM. You mentioned that you can run this with `cargo run --bin ndc-postgres-cli -- <args>`. Would be nice if as a follow-up this could be added as a `just` command and the development.md doc could be updated accordingly as well.

File: crates/cli/tests/update_tests.rs (hunk 1 above)
Comment (SamirTalwar) [reply crates/cli/tests/update_tests.rs:10]: Yeah, it's a normal test case; connects to the database just like the query tests do.

**Comment by SamirTalwar** [comment]
@soupi: I'm gonna update the scripts right now, I just wanted to do it in a separate PR to make review easier. 😁

**Pull request merged**